### DISCUSSION TITLE
FileTypeUtil.getType 为什么不在方法内部还原流指针？

### DISCUSSION THREAD

**Discussion by HakureiMarisa** [looly/hutool#2331] [Q&A] [answered]
https://github.com/dromara/hutool/issues/2237 

如issues所述，既然方法内部会读取28个字节，为什么判断好后不还原呢，让调用者自己mark、reset？

**Comment by looly** [comment, marked as answer]
因为你无法确定用户提供的流是否支持mark和reset，只能由用户来操作了。
